src/actions/expandHoverDown.ts

```diff
@@ -11,7 +11,6 @@ import hashPath from '../util/hashPath'
 import head from '../util/head'
 import pathToContext from '../util/pathToContext'
 
-// eslint-disable-next-line prefer-const
```

Comment:
?

Reply:
Basically as per eslint v9.

if there is no rule define in eslint.config.ts and we are adding the command in our code. so eslint lint will show that command as warning.

so we just fix all these by running yarn lint --fix. if you want to keep these warning as it is so will revert all these.

Reply:
we have added the prefer const to rule in new eslint config now warning is removed.